packages/deprecation-crawler/sandbox/deprecations/all-lowercase.md

```diff
@@ -1,9 +1,9 @@
 <!-- ruid-groups
 
-- master:
-  - /tree/master/all-lowercase/crawled.ts#L12
-  - /tree/master/all-lowercase/crawled.ts#L18
-  - /tree/master/all-lowercase/crawled.ts#L24
+- :
```

Comment:
I just see this, this probably has to be fixed 🤔
But this also raises the question if untagged versions should be included in the markdown output files.

Reply:
Nice pr! I'll have a think on that!

Reply:
I thought about giving it the "version" of the main branch until there the next tag arrives?

This most probably introduces more fiddly logic, so I'm not sure if this is a good idea,.
But at least we have the conversation about the problem ;)

Reply:
I think that's maybe too complex for now 😅

The current implementation of the group markdowns is that it will append new deorecations to the file, correct (meaning that you could make manual edits to the existing deprecations in the file, and it won't get overwritten)?


I also have a second question, does it make sense to have the `checkout` step?
I mean, if you go to a different branch, and you pass in a next version, that seems a little bit odd to me.
In my opinion (I could be wrong), we don't need the checkout step when we run the crawler in its normal mode. It will just crawl the current branch. 
If we implement the history command, that's the command where we should use the checkout step (possibly multiple times).

Reply:
> The current implementation of the group markdowns is that it will append new deprecations to the file, correct (meaning that you could make manual edits to the existing deprecations in the file, and it won't get overwritten)?

It went to the top of the file and replaces the whole comment block with all the deprecations form raw-depreciations.
So whatever is done in the file stays untouched, but the comments are completely new. All edits in the comments are lost which is no problem as comments.. :D

Reply:
> I also have a second question, does it make sense to have the checkout step?

No, but the last valid tag should get determined right? 

How does it work in master? I can imagine in the output formatted we can replace empty versions with the main name?

Reply:
Oh I see, you've merged this one in.
Just as an fyi, I've taken a closer look to the generated markdown and I agree with you 😁
I will create a follow up PR to add the main branch 😉